lib/medera.ex

```diff
@@ -39,4 +28,30 @@ defmodule Medera do
     Endpoint.config_change(changed, removed)
     :ok
   end
+
+  def child_specs(true) do
+    slack_children() ++ web_children() ++ minion_children()
```

Comment:
Yes I think there would be some applications started (Phoenix, Ecto, etc).  I think the minion would need some refinement in how it's packaged before this would be something easy to deploy.